lib/grpcApi/handlers/startTransactionHandlerFactory.js

```diff
@@ -0,0 +1,27 @@
+const { StartTransactionResponse } = require('@dashevo/drive-grpc');
+
+const InternalGrpcError = require('../error/InternalGrpcError');
+
+/**
+ * @param {MongoDBTransaction} stateViewTransaction
+ * @returns {startTransactionHandler}
+ */
+module.exports = function startTransactionHandlerFactory(stateViewTransaction) {
+  /**
+   * Start new transaction in database
+   *
+   * @typedef startTransactionHandler
+   * @returns {Promise<StartTransactionResponse>}
+   */
+  async function startTransactionHandler() {
+    try {
+      stateViewTransaction.start();
+    } catch (error) {
+      throw new InternalGrpcError(error);
```

Comment:
It's not an internal error. It's misusing of the API method.

Reply:
It's misusing only if we have `TransactionIsAlreadyStartedError`.